conftest.py

```diff
@@ -15,3 +20,29 @@ def pytest_collection_modifyitems(config, items):
     for item in items:
         if "latest_runtime" in item.keywords:
             item.add_marker(skip_latest)
+
+
+@pytest.fixture
+def software():
+    return os.environ.get(
+        "COILED_SOFTWARE_NAME",
+        f"dask-engineering/coiled_dist-py{sys.version_info[0]}{sys.version_info[1]}",
+    )
+
+
+@pytest.fixture
+def small_cluster(software):
+    with Cluster(
+        software=software,
+        account="dask-engineering",
+        n_workers=10,
+        worker_vm_types=["m5.large"],
+        scheduler_vm_types=["m5.large"],
```

Comment:
```suggestion
        scheduler_vm_types=["m5.large"],
        wait_for_workers=True,
```
Debatable. Leaving it out increases chaos. Adding it increases determinism. Looks like the default is 0.3.

Reply:
This repository is used both for stability and for performance testing.  My guess is that we're going to want to split this out pretty soon to different options, but for now I don't care that much.